### ISSUE TITLE
Make use of ObservableCollection.AddRange (when it exists)

### ISSUE THREAD

**Issue by bender2k14** [elmish/Elmish.WPF#232]
`ObservableCollection` does not currently have bulk add support, but [Microsoft plans to add it](https://github.com/dotnet/runtime/issues/18087).  We could make some minor improvements with this.

Current milestone is .NET 5.  On the other hand, the issue was created almost four years ago.

Other than implementing this ourselves (which I don't want to do atm), we must wait until this has been released.  In the meantime, I don't mind closing this issue.

**Comment by cmeeren** [comment]
Good call. But if only available in .NET 5, I guess we need to multitarget and do some conditional compilation.

>  In the meantime, I don't mind closing this issue.

Your call, it's all the same to me. :)

**Comment by Evangelink** [comment]
> So definitely not trying to hijack this thread, but for anyone who's in immediate need of this for production code, there is an alternative in [DynamicData](https://github.com/reactivemarbles/DynamicData)'s [SourceCache<T>](https://dynamic-data.org/2015/08/12/observable-cache-at-a-glance/) (_preferred_) and [SourceList<T>](https://dynamic-data.org/2015/07/27/observable-list-at-a-glance/).

If you expect the performance improvement to be significant maybe considering some objects from `ReactiveUI` and `DynamicData` could help.
